test/testcafe/framework/shared/index.js

```diff
@@ -84,6 +84,13 @@ export async function assertRequestMatches(loggedRequest, url, method, body) {
   }
 }
 
+// eslint-disable-next-line @okta/okta/no-unlocalized-text-in-templates
+export const OktaDashboardContent = `
```

Comment:
nit: I think only classes with constructors should be exported as TitleCase
```suggestion
export const oktaDashboardContent = `
```